euth/projects/templates/euth_projects/includes/project_tile.html

```diff
@@ -17,13 +18,13 @@
         {% endif %}
     </a>
     <div class="project-tile-body">
-        <h2 class="project-tile-title"><a href="{% url 'project-detail' project.slug %}">{{ project }}</a></h2>
+        <h2 class="project-tile-title"><a href="{% url 'project-detail' project.slug %}" {% if example %}target="_blank"{% endif %}>{{ project }}</a></h2>
```

Comment:
I find the `example` name for this flag not optimal, because it doesn't describe what this does. I can't come up with a better alternative right now (maybe `open_in_new_tab`).